content/blog/2021/machine-learning.md

```diff
@@ -0,0 +1,58 @@
+Title: Machine learning: A major cause of the Pythonic buzz
+Date: 2021-08-24 12:05
+Author:
+Category: 2021
+Slug: machine-learning-pycon
+Summary: What is machine learning? Where does Python play its part? What's in store for you at PyCon India 2021 for Machine Learning?
+
+## Machine learning: A major cause of the Pythonic buzz
+
+ **“Predicting the future isn’t magic, it’s artificial intelligence.” ~Dave Waters**
+
+![Relations between AI, ML, DL, Big Data and Data Science]({static}/images/ml-article-2021-Relations.png)
+(Image credits: https://miro.medium.com/max/650/1*-XKVI5SAEpffNR7BusdvNQ.png)
```

Comment:
Can we move Image Credits at the bottom of the blog?

Reply:
Can I please know where exactly I need to mention it? It will be really helpful then.